src/colorizer/Dataset.ts

```diff
@@ -449,16 +453,18 @@ export default class Dataset {
     }
 
     // Keep original sorting order of features by inserting in promise order.
-    featureResults.forEach((result) => {
-      const featureValue = this.getPromiseValue(result, "Failed to load feature: ");
+    featureResults.forEach((result, index) => {
+      const featureValue = this.getPromiseValue(result, `Failed to load feature ${index}`);
```

Comment:
Not included in this PR, but `getPromiseValue` looks a lot more like a util function than a class method to me. It doesn't look like it refers to `this` anywhere, though perhaps it might in the future?

Reply:
I don't expect it would ever need to use class-related content... I'll move it